cmd/nginx-ingress/flags.go

```diff
@@ -66,6 +66,9 @@ var (
 	appProtectDosMaxWorkers = flag.Int("app-protect-dos-max-workers", 0, "Max number of nginx processes to support. Requires -nginx-plus and -enable-app-protect-dos.")
 	appProtectDosMemory     = flag.Int("app-protect-dos-memory", 0, "RAM memory size to consume in MB. Requires -nginx-plus and -enable-app-protect-dos.")
 
+	agent              = flag.Bool("agent", false, "Enable NGINX Agent")
```

Comment:
do we need to add these on https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/?